lib/grpcServer/handlers/createGrpcErrorFromDriveResponse.js

```diff
@@ -0,0 +1,79 @@
+const cbor = require('cbor');
+const {
+  server: {
+    error: {
+      InternalGrpcError,
+      InvalidArgumentGrpcError,
+      DeadlineExceededGrpcError,
+      ResourceExhaustedGrpcError,
+      NotFoundGrpcError,
+      FailedPreconditionGrpcError,
+      UnavailableGrpcError,
+      GrpcError,
+    },
+  },
+} = require('@dashevo/grpc-common');
+const GrpcErrorCodes = require('@dashevo/grpc-common/lib/server/error/GrpcErrorCodes');
+const createConsensusError = require('@dashevo/dpp/lib/errors/consensus/createConsensusError');
+const AlreadyExistsGrpcError = require('@dashevo/grpc-common/lib/server/error/AlreadyExistsGrpcError');
+
+/**
+ * @typedef createGrpcErrorFromDriveResponse
+ * @param {number} code
+ * @param {string} info
+ * @return {GrpcError}
+ */
+function createGrpcErrorFromDriveResponse(code, info) {
+  if (code === undefined) {
+    return new InternalGrpcError(new Error('Drive’s error code is empty'));
+  }
+
+  const decodedInfo = cbor.decode(Buffer.from(info, 'base64'));
+
+  // eslint-disable-next-line default-case
+  switch (code) {
+    case GrpcErrorCodes.INVALID_ARGUMENT:
+      return new InvalidArgumentGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.DEADLINE_EXCEEDED:
+      return new DeadlineExceededGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.NOT_FOUND:
+      return new NotFoundGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.ALREADY_EXISTS:
+      return new AlreadyExistsGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.RESOURCE_EXHAUSTED:
+      return new ResourceExhaustedGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.FAILED_PRECONDITION:
+      return new FailedPreconditionGrpcError(decodedInfo.message, decodedInfo.metadata);
+    case GrpcErrorCodes.INTERNAL: {
+      const error = new Error(decodedInfo.message);
+      error.stack = decodedInfo.metadata.stack;
+
+      delete decodedInfo.metadata.stack;
```

Comment:
Theoretically, if verbose errors are not enabled in Drive `decodedInfo.metadata.stack` will be undefined. Probably we should check if the stack is exists in metadata.